pandas/tests/window/test_rolling.py

```diff
@@ -488,7 +488,7 @@ def test_rolling_axis_count(axis_frame):
 
     axis = df._get_axis_number(axis_frame)
 
-    if axis in [0, "index"]:
+    if DataFrame()._get_axis_number(axis)==0:
```

Comment:
`axis` has already been reduced to an integer in line 489. Sufficient for `if axis == 0`

Reply:
Ah. So, has this specific edit been rendered unnecessary?